src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/ToolStripMenuItemTests.cs

```diff
@@ -183,6 +185,225 @@ public static IEnumerable<object[]> CultureInfo_Shortcut_TestData()
         yield return new object[] { new CultureInfo("de-DE"), new CultureInfo("de-DE"), "Strg+Umschalttaste+K" };
     }
 
+    [WinFormsFact]
+    public void ToolStripMenuItem_Ctor_ShouldInitializeCorrectly()
+    {
+        using Bitmap image = new(10, 10);
+        const string text = "Test Item";
```

Comment:
Normally const are defined at the beginning of a class. We don't need to specify one in the method.
```suggestion
        string text = "Test Item";
```